test/server/middleware/executor/Executor.spec.js

```diff
@@ -3,331 +3,456 @@
  * @author pmeijer / https://github.com/pmeijer
  */
 
-var testFixture = require('../../../_globals.js');
+const testFixture = require('../../../_globals.js');
```

Comment:
i will need to check our coverage and I am assuming we completely backward compatible...

Reply:
Yeah, it should be backwards compatible. In the ExecutorMaster, I just added a method to add the user to the mongo query if executor authentication is enabled. Otherwise, it is just a no-op and it uses the original query.